modules/distribution/carbon-home/resources/dashboards/rdbms-statistics/WSO2 Streaming Integration - RDBMS Table Statistics.json

```diff
@@ -1227,7 +1313,7 @@
     ]
   },
   "timezone": "",
-  "title": "WSO2 Streaming Integrator - RDBMS Table Statistics",
+  "title": "WSO2 Streaming Integration - RDBMS Table Statistics",
```

Comment:
Why this is changed to Streaming Integration since the product is Streaming Integrator

Reply:
Resolve in 094705475651d37ee5e03f75532cad40a82e7d82